### PR TITLE
Fix most checkstyle errors

### PR DESCRIPTION
**Pull request by whyjayxp**
The remaining checkstyle errors have to do with missing javadocs and visibility of attributes. Do fix the ones in your own component/features.

### CODE REVIEW

File: src/main/java/seedu/address/model/UserPrefs.java [hunk 1]
```diff
@@ -100,11 +101,9 @@ public boolean equals(Object other) {
...
         UserPrefs o = (UserPrefs) other;
 
-        return guiSettings.equals(o.guiSettings)
-                && menuFilePath.equals(o.menuFilePath)
-                && ordersFilePath.equals(o.ordersFilePath)
-                && tablesFilePath.equals(o.tablesFilePath)
-                && statisticsFilePath.equals(o.statisticsFilePath);
+        return guiSettings.equals(o.guiSettings) && menuFilePath.equals(o.menuFilePath) && ordersFilePath
+                .equals(o.ordersFilePath) && tablesFilePath.equals(o.tablesFilePath) && statisticsFilePath
+                .equals(o.statisticsFilePath);
```
Comment (Qing-Yuan) [inline src/main/java/seedu/address/model/UserPrefs.java]: This line should be indented one more level

File: src/main/java/seedu/address/model/menu/MenuItem.java [hunk 2]
```diff
@@ -96,11 +93,7 @@ public int hashCode() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append(getName())
-                .append(" Code: ")
-                .append(getCode())
-                .append(" Price: ")
-                .append(getPrice());
+        builder.append(getName()).append(" Code: ").append(getCode()).append(" Price: ").append(getPrice());
```
Comment (Qing-Yuan) [inline src/main/java/seedu/address/model/menu/MenuItem.java]: Even though this edit does not violate any conventions, I believe the previous form was more readable

File: src/main/java/seedu/address/model/person/Person.java [hunk 3]
```diff
@@ -105,14 +102,8 @@ public int hashCode() {
...
-        builder.append(getName())
-                .append(" Phone: ")
-                .append(getPhone())
-                .append(" Email: ")
-                .append(getEmail())
-                .append(" Address: ")
-                .append(getAddress())
-                .append(" Tags: ");
+        builder.append(getName()).append(" Phone: ").append(getPhone()).append(" Email: ").append(getEmail())
+                .append(" Address: ").append(getAddress()).append(" Tags: ");
```
Comment (Qing-Yuan) [inline src/main/java/seedu/address/model/person/Person.java]: Similarly, I believe the original is more reader friendly

File: src/main/java/seedu/address/model/statistics/Bill.java [hunk 4]
```diff
@@ -50,13 +51,8 @@ public int hashCode() {
...
         final StringBuilder builder = new StringBuilder();
-        builder.append("[Table ")
-                .append(getTableNumber())
-                .append("] [Total Cost of the Bill: $")
-                .append(getTotalBill())
-                .append("] [Date: ")
-                .append(getFormattedDate())
-                .append("] ");
+        builder.append("[Table ").append(getTableNumber()).append("] [Total Cost of the Bill: $").append(getTotalBill())
+                .append("] [Date: ").append(getFormattedDate()).append("] ");
```
Comment (Qing-Yuan) [inline src/main/java/seedu/address/model/statistics/Bill.java]: Same as previous comments

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 5]
```diff
@@ -35,8 +35,8 @@
      */
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
-            @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+                             @JsonProperty("email") String email, @JsonProperty("address") String address,
+                             @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
```
Comment (Qing-Yuan) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java]: Just to clarify is this follows Java conventions, it seems like a special case

File: src/main/java/seedu/address/model/UserPrefs.java (hunk 1 above)
Comment (whyjayxp) [reply src/main/java/seedu/address/model/UserPrefs.java]: This complies with the Java Coding Standard.

File: src/main/java/seedu/address/model/person/Person.java (hunk 3 above)
Comment (whyjayxp) [reply src/main/java/seedu/address/model/person/Person.java]: This complies with the Java Coding Standard.

**Pull request merged**